bindings/matrix-sdk-ffi/src/platform.rs

```diff
@@ -358,6 +358,41 @@ pub fn setup_tracing(config: TracingConfiguration) {
         .init();
 }
 
+/// Set up a lightweight tokio runtime, for processes that have memory
+/// limitations (like the NSE process on iOS).
+#[matrix_sdk_ffi_macros::export]
+pub fn setup_lightweight_tokio_runtime() {
+    async_compat::set_runtime_builder(Box::new(|| {
+        eprintln!("spawning a lightweight tokio runtime");
+
+        // Get the number of available cores through the system, if possible.
+        let num_available_cores =
+            std::thread::available_parallelism().map(|n| n.get()).unwrap_or(1);
+
+        // The number of worker threads will be either that or 4, whichever is smaller.
+        let num_worker_threads = usize::min(num_available_cores, 4);
```

Comment:
Why not `cmp:min()`? I think this one is expected to be used as `a.min(b)`, at least that's what the examples show.

Reply:
my mathematical brain wants `function(param1, param2)` instead of `param1.function(param2)`, in this case, but both work.